### PR TITLE
Add sponsorship from @Urigo

### PR DESCRIPTION
**Pull request by hayes**
@Urigo thanks so much for the sponsorship!  This PR is a placeholder for adding you to the sponsors list.  Any preference on what I add here (github name, real name, The Guild logo, etc?)

### CODE REVIEW

**Comment by Urigo** [comment]
@hayes could you change the reference or just add The Guild account?
https://github.com/the-guild-org
Thank you for all your wonderful work!

**Comment by Urigo** [comment]
adding the guild logo would be awesome!

**Comment by hayes** [comment]
@huv1k Added you to the sponsorships here as well, let me know if you want me to change how you are listed, or what page I link to.

File: README.md [hunk 1]
```diff
@@ -114,5 +114,12 @@ server.start();
...
 
+## Sponsors
+
+Pothos development supported by [sponsorships](https://github.com/sponsors/hayes) from these
+generous people and organizations:
+
+- [![The Guild](https://pothos-graphql.dev/assets/the-guild-logo.svg)](https://www.the-guild.dev/)
 - [@kidqueb](https://github.com/kidqueb)
 - [@homoky](https://github.com/homoky)
+- [@huv1k](https://github.com/https://github.com/huv1k)
```
Comment (huv1k) [inline README.md:125]: Here is t wice GitHub link 😅

Comment (hayes) [reply README.md:125]: Whoops, fixed

Comment (hayes) [reply README.md:125]: Also, thank you for the sponsorship!

Comment (huv1k) [reply README.md:125]: I thank you for great framework 🙌  https://mobile.twitter.com/huv1k/status/1514500281823862784

**Pull request merged**